infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/VerticalGeometryListing.kt

```diff
@@ -313,100 +316,91 @@ fun toVerticalGeometryListing(
-        it.end.height
-    },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.ANGLE_END)) {
-        it.end.angle
-    },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.LOCATION_E_END)) { it.end.location?.roundedX },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.LOCATION_N_END)) { it.end.location?.roundedY },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.RADIUS)) { it.radius },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.TANGENT)) {
-        it.tangent
-    },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.LINEAR_SECTION_BACKWARD_LENGTH)) {
-        it.linearSectionBackward.stationValueDistance
-    },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.LINEAR_SECTION_BACKWARD_LINEAR_SECTION)) {
-        it.linearSectionBackward.linearSegmentLength
-    },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.LINEAR_SECTION_FORWARD_LENGTH)) {
-        it.linearSectionForward.stationValueDistance
-    },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.LINEAR_SECTION_FORWARD_LINEAR_SECTION)) {
-        it.linearSectionForward.linearSegmentLength
-    },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.STATION_START)) { it.start.station },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.STATION_POINT)) { it.point.station },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.STATION_END)) { it.end.station },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.VERTICAL_COORDINATE_SYSTEM)) { it.verticalCoordinateSystem },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.ELEVATION_MEASUREMENT_METHOD)) {
-        translateElevationMeasurementMethod(it.elevationMeasurementMethod)
-    },
-    CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.REMARKS)) { if (it.overlapsAnother) VERTICAL_SECTIONS_OVERLAP else "" }
-)
+private fun commonVerticalGeometryListingCsvEntries(translation: Translation): List<CsvEntry<VerticalGeometryListing>> =
+    mapOf<String, (item: VerticalGeometryListing) -> Any?>("$VERTICAL_GEOMETRY_CSV_TRANSLATION_PREFIX.plan-name" to { it.fileName },
+        "$VERTICAL_GEOMETRY_CSV_TRANSLATION_PREFIX.crs" to { it.coordinateSystemSrid ?: it.coordinateSystemName },
+        "$VERTICAL_GEOMETRY_CSV_TRANSLATION_PREFIX.creation-date" to { it.creationTime?.let(::toCsvDate) },
+        "$VERTICAL_GEOMETRY_CSV_TRANSLATION_PREFIX.plan-track" to { it.alignmentName },
+        "$VERTICAL_GEOMETRY_CSV_TRANSLATION_PREFIX.track-address-start" to {
+            it.start.address?.let { address ->
+                formatTrackMeter(
```

Comment:
Tuo formatTrackMeter on aika erikoinen signature kun se nimensä mukaisesti formatoi trackMeterin mutta sille ei voi antaa sisään trackMeteriä vaan täytyy antaa sen sisältä molemmat kentät. En tiedä tarvitseeko joku oikeasti tuollaista muotoa, mutta veikkaisin että 99% käyttötapauksista on nimenomaa TrackMeter-tyyppi joka pitäisi formatoida. Eli vähintään voisi tehdä overloadin ja ehkä jopa tämän nykyversion poistaa jos sitä kukaan ei oikeasti tarvitse. Silloin myös tämä käyttöpaikka menee simppelimmäksi: `it.start.address?.let(::formatTrackMeter)`